### DISCUSSION TITLE
Is it possible to assign camera position from an external gps device?

### DISCUSSION THREAD

**Discussion by SECOPtopo** [AR-js-org/AR.js#513] [Help]
I am developing an application where I get the actual position from an external gps device (emlid reach m2) linked to my phone. Is it possible to assign the location obtained from the external GPS as the location of my mobile in AR.js?
Thanks in advance

**Comment by nickw1** [comment]
I'm not sure but I would guess this is a phone configuration issue, if you configure your phone to obtain location from the external device then AR.js will use that location.

This is only a guess so if anyone more knowledgeable has some other info, please feel free to say.
